UnityProject/Assets/Scripts/Objects/GasContainer.cs

```diff
@@ -38,10 +38,18 @@ public class GasContainer : NetworkBehaviour, IGasMixContainer, IServerSpawn
 
 		private bool gasIsInitialised = false;
 
+		private Pickupable pickupable;
+
+		[SyncVar]
+		//Only updated and valid for canisters inside the players inventory!!!
+		private float oxygenRatio = 0;
+		public float OxygenRatio => oxygenRatio;
```

Comment:
feels hacky, shouldnt this be on the players UI instead of each gascontainer? 
Small note, its name is oxygenratio while in reality it's just the relation of the max capacity and current moles

Reply:
cant put a syncvar on UI, would have to do two netmessages instead. If you think thats better?